spec/commands/creator_spec.rb

```diff
@@ -1,27 +1,29 @@
-require 'pliny/commands/creator'
-require 'spec_helper'
+require "pliny/commands/creator"
```

Comment:
Is there any chance you could split the formatting/style changes out into separate commits in future? It would make seeing the actual real changes a lot easier.

Reply:
Yes, I can be more careful about how I build the commits via `git add -p`. My tooling (Vim + Ale + standard.rb/rubocop) basically take care of the syntax stuff for me these days, so I've gotten a bit lax about it. I'll try to keep them discrete going forward.

Reply:
Thanks 😄Yeah it does add a bit of faff when making the commits but hopefully we can agree it's worth it to keep the changes clear 👍